components/Multiselector.jsx

```diff
@@ -0,0 +1,111 @@
+import { useRef } from "react";
+import Multiselect from "multiselect-react-dropdown";
+import styled from "styled-components";
+import { baseTheme } from "../theme";
+
+/*
+    params: 
+    - options = available items to choose from
+    - displayValue = property name to display in the dropdown options
+    - placeholder = placeholder in the input section
+    - setList = function that will update the listType variable (holds all valid tags)
+    - searchValue = current string of search field input
+    - tagValue = current array of selected tags
+    - otherValue = current array of selected values (blog --> releaseBatch, opp --> city)
+    - handleChange = function in main jsx file that will execute every change 
+    - isFilterTag = boolean indicating if the filter is for tags or otherValue
+*/
+export const Multiselector = ({
+  options,
+  displayValue,
+  placeholder,
+  setList,
+  searchValue,
+  tagValue,
+  otherValue,
```

Comment:
This is acceptable for now but really shouldn't be done. Instead you can have one state object passed and it updates an object:
const [sampleValues, setSampleValues] = useState({
    search: "",
    filter: "",
    tags: ""
});

and you pass sampleValues through so there aren't infinite props but this can be changed in the future

Reply:
ok, I will leave this for now then